packages/reaction-orders/server/register.js

```diff
@@ -59,34 +59,41 @@ ReactionCore.registerPackage({
     label: "Order Processing",
     status: "created",
     workflow: "coreOrderWorkflow",
-    audience: ["dashboard/orders"]
+    audience: ["dashboard/orders"],
+    emailTemplates: ["orders/new"]
```

Comment:
Sent from mobile device. Please excuse brevity and typos.
On Jun 6, 2016 11:50 PM, "Aaron Judd" <notifications@github.com> wrote:
>
> In packages/reaction-orders/server/register.js:
>
> > @@ -59,34 +59,41 @@ ReactionCore.registerPackage({
> >      label: "Order Processing",
> >      status: "created",
> >      workflow: "coreOrderWorkflow",
> > -    audience: ["dashboard/orders"]
> > +    audience: ["dashboard/orders"],
> > +    emailTemplates: ["orders/new"]
>
> Previously the logic just used let tpl =orders/${order.workflow.status};,
and didn't need a emailTemplates... do you think it's better to have to
define a template for each, or to just assume that every workflow state
that supports a template, does has the emailTemplate assignment?

I'm totally open to that. I'm not even sur if we want to have the templates
coupled to workflow steps/UI containers. it could make sense if we plan to
automatically trigger notification when a certain state is entered.

>
> —
> You are receiving this because you commented.
>
> Reply to this email directly, view it on GitHub, or mute the thread.